src/v2i-hub/TimPlugin/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+	"name":"TIM",
+	"description":"Provides Traveller Information Message (TIM).",
+	"version":"3.2.0",
+	"exeLocation":"/bin/TimPlugin",
+	"coreIpAddr":"127.0.0.1",
+	"corePort":24601,
+	"messageTypes":[
+		{
+			"type":"J2735",
+			"subtype":"TIM",
+			"description":"Travelor Information Message indicating the active Work Zone information."
+		}
+	],
+	"configuration":[
+		{
+			"key":"Frequency",
+			"default":"1000",
+			"description":"The frequency to send the TIM in milliseconds."
+		},
+		{
+			"key":"MapFile",
+			"default":"IVP_GF_TIM.xml",
+			"description":""
+		},
+		{
+			"key":"Snap Interval",
+			"default":"300",
+			"description":"The interval in milliseconds to keep a vehicle within a zone before allowing it to transition out of all zones."
+		},
+		{
+			"key":"Vehicle Timeout",
+			"default":"2000",
+			"description":"Timeout in milliseconds when a vehicle is removed from all zones if a BSM has not been received."
+		},
+		{
+			"key":"Start Broadcast Date (mm-dd-YYYY)",
+			"default":"01-01-2019",
+			"description":"The Start Broadcast Date for the TIM message in the (mm-dd-YYYY) format."
+		},
+		{
+			"key":"Stop Broadcast Date (mm-dd-YYYY)",
+			"default":"12-31-2019",
+			"description":"The Stop Broadcast Date for the TIM message in the (mm-dd-YYYY) format."
+		},
+		{
+			"key":"Start Broadcast Time (HH:MM:SS)",
```

Comment:
How are broadcast start date/time managed when there are more than one TIM message that require to be broadcast?  Also, when the same TIM message needs to be broadcast at different times during the day, how can that be configured?

Reply:
Config values through GUI are mainly for testing. I have created a function TimDuration(), that pulls in values from GetConfigValues to ensure Tim broadcast within a specific duration. We need to create another function to take input for multiple Tim messages and pass it to the TimDuration function as arguments. This can be done once the format of message coming from web-service is finalized.